### PR TITLE
Add inline audio example

### PR DESCRIPTION
**Pull request by MarkDaoust**

### CODE REVIEW

File: quickstarts/Audio.ipynb [hunk 1]
```diff
@@ -17,27 +17,37 @@
     },
```
Comment (random-forests) [inline quickstarts/Audio.ipynb:17]: <pre class="ql-syntax" spellcheck="false">Line #2.    from IPython import display
</pre><p>Maybe nix this import and the display.Markdown below? Just for simplicity.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/80/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (random-forests) [inline quickstarts/Audio.ipynb:17]: <p>Can we just say "Attach it to the prompt in the list of parts." (omitting the glm.Blob detail, I don't see that mentioned in the code - looks like it works automagically). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/80/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Comment (random-forests) [reply quickstarts/Audio.ipynb:17]: And, thank you for adding this!

Comment (MarkDaoust) [reply quickstarts/Audio.ipynb:17]: <p>Done.</p>

Comment (MarkDaoust) [reply quickstarts/Audio.ipynb:17]: <p>Done.</p>

**Review by shilpakancharla — APPROVED** [review]

**Pull request merged**